integration/integrate-with-pagerduty.adoc

```diff
@@ -0,0 +1,22 @@
+[id="integrate-with-pagerduty"]
+= Integrating with PagerDuty
+include::modules/common-attributes.adoc[]
+:context: integrate-with-pagerduty
+
+toc::[]
+
+[role="_abstract"]
+If you are using https://www.pagerduty.com/[PagerDuty], you can forward alerts from {product-title} to PagerDuty.
+
+High-level integration flow:
```

Comment:
Would you mind reusing the modified text in the "Integrating with Slack" assembly? 
> The following steps represent a high-level workflow for integrating Red Hat Advanced Cluster Security for Kubernetes with PagerDuty:

Reply:
Yeah, only because I created these PRs all together. I'll check and update them all.